File: Practice/11/C++/example11/example11/example11.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    setlocale(LC_ALL, "Russian");

    double ex, result = 1;
    int n;
    cin >> ex;
    cin >> n;
    if (n >= 0) {
        for (int i = 0; i < n; i++)
        {
            result = result * ex;
        }
    }
    else {
        for (int i = 0; i < -n; i++)
        {
            result = result * ex;
        }
        result = 1 / result;
    
    }

    cout << result;



}

File: Practice/11/Python/example11.py
result = 1;
ex = float(input())
n = int(input())
i=0
if n >= 0: 
	while i < n:
		i += 1
		result = result * ex
else:
	while i < -n:
		i += 1
		result = result * ex
	result = 1/result


print(result)


